src/shell/shell.cpp

```diff
@@ -1258,6 +1260,31 @@ void SHELL_Init() {
 	        " Volume in drive %c is %s\n"
 	        " Volume Serial Number is %04X-%04X\n"
 	        "\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP",
+	        "Moves files and renames files and directories\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP_LONG",
+	        "Usage:\n"
+	        "To move one or more files:\n"
+	        "  [color=green]move[reset] [color=cyan][drive:][path]filename1[,...] destination[reset]\n"
+	        "\n"
+	        "To rename a directory:\n"
+	        "  [color=green]move[reset] [color=cyan][drive:][path]dirname1 dirname2[reset]\n"
+	        "Where:\n"
+	        "  [color=white]filename[reset]      Can be either an exact filename or an inexact filename with\n"
+	        "              wildcards, which are the asterisk (*) and the question mark (?).\n"
+	        "              Multiple tokens can be provided and must be seperated by commas.\n"
+	        "  [color=cyan]destination[reset] An exact filename or directory, not containing any wildcards.\n"
+	        "  [color=cyan]dirname[reset] An exact directory name, not containing any wildcards.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  If multiple source files are specified, destination must be a directory (or one will be created for you).\n"
+	        "  If a single source file is specified, it will overwrite destination, if it exists.\n"
+	        "\n"
+	        "Examples:\n"
+	        "  [color=green]move[reset] [color=white]source.bat[reset] [color=cyan]new.bat[reset]\n"
+	        "  [color=green]move[reset] [color=white]file1.txt,file2.txt[reset] [color=cyan]mydir[reset]\n");
+	MSG_Add("SHELL_CMD_MOVE_MULTIPLE_TO_SINGLE",
+	        "Cannot move multiple files to a single file\n");
```

Comment:
- Indentation after `filename` is off
- Lines in the notes section are too long
- Insert blank line above `Where:`

<img width="1215" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/ba5f2407-bb04-4761-b4e8-89ae08118d6f">

Reply:
Fixed